src/lib.rs

```diff
@@ -1,10 +1,13 @@
-#![forbid(unsafe_code, future_incompatible)]
```

Comment:
Prefer if this was instead moved into `deny`?

Reply:
Future incompatible is warn by default and that seems reasonable. I've been trying to standardize my default set of lints across projects, so this was just pasting in my updated version. I can't remember why I took out forbid future incompatible at some point

Reply:
Probably because of https://github.com/rust-lang/rust/issues/81670